bsps/shared/start/stackalloc.c

```diff
@@ -43,15 +44,19 @@ void bsp_stack_allocate_init(size_t stack_space_size)
 void *bsp_stack_allocate(size_t size)
 {
   void *stack = NULL;
-
+  uint32_t *page_table_base;
```

Comment:
That's a shared code between all RTEMS ports. `uint32_t` won't be correct for 64-bit architecture. It's better to make it `uintptr_t`